### DISCUSSION TITLE
No Github release since v24.7.17

### DISCUSSION THREAD

**Discussion by maurice-audin** [jgraph/drawio#4778] [General]
Hello,

There is no Github release since v24.7.17: https://github.com/jgraph/drawio/releases. My upgrade process was based on these releases, are you planing on using them again or should I now build the .war file ? If so, do you have a doc describing the build process ?

Thanks!

**Comment by davidjgraph** [comment]
we have some issues with the build file that we're working on.

**Comment by maurice-audin** [comment]
Ok, thanks for the update and the good work on drawio !
